client/examples/http_client.rs

```diff
@@ -515,7 +530,9 @@ async fn build_client(
         FallbackCryptoProvider::default()
     };
 
-    let mut builder = ClientBuilder::from_url(relay_addr).crypto(provider);
+    let mut builder = ClientBuilder::from_url(relay_addr)
+        .crypto(provider)
+        .expire_session_after(std::time::Duration::from_secs(20));
```

Comment:
Maybe this value should be a CLI param
https://github.com/golemfactory/ya-relay/blob/8f308fa9f00650ef674e1eae361344f79dd4b3ba/client/examples/http_client.rs#L36

Reply:
Yes, but on the other hand it's just an example, making it too complicated doesn't help with the usage

Reply:
Ok.